### DISCUSSION TITLE
[Factions] Faction Outposts punish player progression

### DISCUSSION THREAD

**Discussion by WJohn** [FakeFishGames/Barotrauma#13443] [Ideas and suggestions]
### The Problem:
Every station is split into one of two Factions. Taking general purpose missions from these stations directly increases your Reputation with that Faction while harming it with the other. It is also by far the best and most reliable way to make money, gain experience, and generally progress.

Because you are first and foremost trying to make money in order to get XP and upgrade your ship, Reputation takes a back seat. This means it hardly matters which station Faction you are going to or what missions you take -- let alone whether you actually agree with the politics of that Faction. You're just trying to make money. Unfortunately, doing this dramatically slows down the progression of the game since you end up in a see-saw of Reputation with neither side rising particularly fast (especially if you're not doing the Politician Captain skill tree). This also makes the impact of griefers in outposts much more severe.

In order to avoid this see-saw of Reputation, you can alternatively avoid taking missions on certain stations as to not harm your Reputation with the Faction you've sided with. This is dependent on RNG for whether outposts are distributed in your favor. Additionally, you lose out on a huge amount of income which you need in order to upgrade your ship to get through the game, and dramatically slows down your character progression since XP is tied to missions.

You are either stuck with poor Reputation on either side and are paying exorbitant prices to get through the Biome gates, or you skip tons of missions at the whim of RNG outpost Faction distribution while remaining very under-powered and under-equipped. **Both fundamental options for progression punish the player.**

### A Possible Solution:
The vast majority of stations, if not all, could be turned neutral. Reputation with most stations now contributes to a global neutral Reputation/karma, and determines your discounts and scrutiny of security searches (but never reaches the point of outright kill on sight for neutral stations when docking, so that missions may still be done). This means that, no matter what, you are at least able to take basic missions (monster nests, wreck clearing, perhaps fighting pirates etc) and still continue progressing through the game without either hurting the Faction Reputation you're going for, or having to skip large amounts of stations because of RNG.

With the stations now being neutral, major Factions could be turned into something more akin to minor Factions -- with Coalition officials in their own area in stations handing out their own unique missions, and Separatists hanging out in the darker, sketchier parts of stations doing much the same. These unique missions differ from the local ones that neutral station Administrators give, instead focusing on more long range missions like Jailbreak or Assassination. Additionally like with the minor Husk and Clown Factions, there should be far more overarching "quest chains" that involve you performing unique missions -- much like how rescuing Jacov Subra is uniquely designed. As these are meant to be major Factions, the variety of missions, length and complexity of quest chains, and number of unique characters for them should proportionally be more richly developed.

### Summary:

1. The current system punishes players no matter how they play, either slowing down their Reputation gain to a crawl simply for trying to make money, or forcing them to skip a lot of missions so they can keep their preferred Reputation high.
2. Stations being neutral and giving neutral Reputation missions would mean you can still earn money and experience without constantly slowing your progression in the campaign, and would let you focus on a specific Faction.
3. Major Factions behaving more like minor Factions in how they're laid out would let players actively choose to stick with them, rather than doing whatever's available out of pure necessity.

Below is a mock-up of what it could look like:
![image](https://github.com/FakeFishGames/Barotrauma/assets/5988156/42b2f47f-fd09-4658-836a-63e386199354)

**Comment by bookslayer10** [comment]
I don't really think this is much of an issue for most lobbies, not to the level that you are describing. Sure, it's a challenge to balance reputation gain with still making money at every outpost. That's a good thing! That means you make tough choices with actual consequences behind them. It's not supposed to be easy to get high reputation while still being able to jobs for every faction. This is the entire reason factions were reworked and (part of) the reason "outpost reputation" was removed.

Like, how would global reputation even work? You do a mission, you gain global reputation - how would it go down? Stealing? Failing missions? That's not interesting or good enough. (Failing missions is already a waste of time, a rep loss would be an infuriating insult to injury.) Global reputation would just be "how long the campaign has been going" meter, there would be nothing interesting. The entire reason there are two competing reputation tracks is because it can actually have a large impact on the game.

I do agree that secondary factions should be shown on the map, but turning all factions into secondary factions is a massive step backwards. I'm fine with players simply not wanting to engage with the system, but the solution to that is modding/cheating, not changes to everyone's game.

**Reply by NotWendy** [reply]
The point of neutral reputation here is mostly to punish you for killing and stealing at neutral outposts. Neutral merchants would offer you basic equipment, but the real good stuff, unique items, and high discounts from reputation would come from those "faction representatives" and fully faction-owned outposts. You can basically see neutral rep as something that wouldn't go into the Friendly/Revered territory at all.

The goal is to let the players consistently earn money for doing missions no matter what, but also incentivize them to pick a side of one of the major factions and stick with it, because neutral missions will only get you so far in terms of equipment you have access to.
Right now it's those low-end missions that allow you to slowly but surely crawl your way up to being liked by both sides, in my experience. Having them not impact your faction reputation lets the higher-end missions with higher impact on reputation balance dictate how the factions view you. You will end up being liked by one side and at war with the other sooner, making your choices feel more impactful. And you won't be forced to pick a side by the RNG of how the outposts generate, if you don't feel forced to take faction-specific missions every single time you visit an outpost, which is how it is right now.

Those are my thoughts on it, at least, as a contributor to the original post. WJohn and Fabevil may have their own ideas.

**Reply by bookslayer10** [reply]
But, like, what if you just don't get caught stealing/killing, through either not doing it or by just being sneaky? Then there wouldn't be any interaction with neutral rep other than it just going up. That seems like an incredible waste of dev time when there already are ways that punish stealing and killing. Seriously, have you ever gotten outpost security mad at you? They have zero chill, ain't no way I'm being caught murdering unless I have a Masterwork HMG and Combat Stimulant.

Anyway, I feel like the current system already has what you are asking for. You are incentivized to play faction-specific missions because they pay more and give you more reputation in one specific faction, but they earn you less net rep overall, or break even. If you don't want to join a side, you can just take the missions that only have positive reputation gain, or at least net-neutral. You will earn more money, and while most of the reputation cancels each other out, some of it won't and will just help you rank up.

There's no easy way to make tons of money while also raising rep quickly - either you only work for one side and skip missions, or you work for both sides and gain rep slowly. That means there are actually engaging choices instead of just grind.

**Comment by EebleSheembl** [comment]
👍 yes
